player/dialog/file_dialog-mac.m

```diff
@@ -0,0 +1,33 @@
+/*
+ * This file is part of mpv.
+ *
+ * mpv is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * mpv is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with mpv.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "file_dialog_detail.h"
+
+#include <osdep/mac/swift.h>
+
+static char **get_files(void *talloc_ctx, const mp_file_dialog_params *params,
+                        bool *error)
+{
+    *error = false;
+    return [[[Dialog alloc] init] openDialog:talloc_ctx params: params];
```

Comment:
my fail here. the space between `params:` and `params` needs to be removed for consistency with other obj-c parts.